src/table/paginator.scss

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+@import './variables';
+
+@mixin iui-paginator {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  align-items: center;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    position: relative;
+    height: 100%;
+    flex: 1;
+  }
+
+  > .iui-center {
+    width: 33%;
+    justify-content: center;
+    @include themed {
+      color: t(iui-text-color);
+    }
+
+    .iui-button-group {
+      margin: 0 $iui-s;
+
+      > .iui-button,
+      > .iui-ellipsis {
+        width: 38px;
+
+        &.iui-small {
+          width: 27px;
```

Comment:
These magic numbers can use variables.

Reply:
[The `$iui-input-height` variables](https://github.com/iTwin/iTwinUI/pull/97/files#r636695492) from #97 would be useful here.  If we can get that PR thru before the table refactor, that would fix this comment as well as [this one](https://github.com/iTwin/iTwinUI/pull/80#discussion_r634702921).

Reply:
I'm going to use `$iui-button-height` for now.

To head off the next obvious question:  why are we using a height variable for the width?  I'm trying to give each button a 1:1 aspect ratio.  I want to avoid examples like below:
![Screen Shot 2021-05-25 at 3 11 57 PM](https://user-images.githubusercontent.com/849817/119555172-8d9da000-bd6b-11eb-9d79-ba6be110a784.png)

But if we get into the very high digits, that will also be an issue.
![Screen Shot 2021-05-25 at 3 13 05 PM](https://user-images.githubusercontent.com/849817/119555411-d6edef80-bd6b-11eb-8483-b03c126acfb9.png)

Maybe another case where we measure the most wide of the bunch and then apply that width to all of them?

Reply:
Would be nice if there's a css-only solution for this